### ISSUE TITLE
[PWA] Is it a good practice to set version in manifest tag?

### ISSUE THREAD

**Issue by gbdematos** [vuejs/vue-cli#4660]
### Version
3.11.0

### Environment info
```
System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-4670 CPU @ 3.40GHz
  Binaries:
    Node: 12.5.0 - C:\Develop\Laragon\bin\nodejs\node-v12\node.EXE
    Yarn: 1.17.3 - C:\Develop\Laragon\bin\yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Develop\Laragon\bin\nodejs\node-v12\npm.CMD
  Browsers:
    Edge: 44.18362.387.0
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
-

### What is expected?
-

### What is actually happening?
-

---
Hi! 

When launching the app if already installed, the manifest is not updating, as seen here:
https://developers.google.com/web/fundamentals/integration/webapks#update-webapk

"Chrome will only update a WebAPK if the Web Manifest URL does not change. If you change the web page from referencing /manifest.json to reference /manifest2.json, the WebAPK will no longer update. (Don't do this!)"

As the assets version is attached to the tag ("manifest.json?v=1.1"), isn't it detected as a different file and not updated? 

I'm having trouble with this, since I have an app and changed it icon, but id didn't update on any of the devices that has it installed.

**Comment by LinusBorg** [comment]
What the docs describe there is that by moving the file to a different path, the webapk don't be able to find it anymore as it can't know about the new path.

But changing a URL parameter isn't the same thing.

As long as your web server doesn't do anything with this `v` parameter (which by default it would not), the webapk will still be able to find the file.

The parameter just forces web browsers to refresh the file as to the browser's cache, this is a new URL.

So in summary: I don't see an issue with this and if you have one ,out would have to provide a reproduction as is *required* in our issue guidelines.

So far the whole problem description is one sentence.

**Comment by gbdematos** [comment]
> What the docs describe there is that by moving the file to a different path, the webapk won't be able to find it anymore as it can't know about the new path.
> 
> But changing a URL parameter isn't the same thing.
> 
> As long as your web server doesn't do anything with this `v` parameter (which by default it would not), the webapk will still be able to find the file.
> 
> The parameter just forces web browsers to refresh the file as to the browser's cache, this is a new URL.
> 
> So in summary: I don't see an issue with this and if you have one, you would have to provide a reproduction as is _required_ in our issue guidelines.
> 
> So far the whole problem description is one sentence.

I don't think this is the case. I added the reproduce steps now, thanks.
